tests/s3_data_plane_tests.c

```diff
@@ -2675,50 +2675,57 @@ static int s_test_s3_round_trip_default_get_fc(struct aws_allocator *allocator,
     struct aws_byte_buf path_buf;
     AWS_ZERO_STRUCT(path_buf);
 
-    ASSERT_SUCCESS(aws_s3_tester_upload_file_path_init(
-        allocator, &path_buf, aws_byte_cursor_from_c_str("/prefix/round_trip/test_default_fc.txt")));
+    /* I know it's ugly, but I don't want to waste time to refact all those flexible checksum tests */
+    for (int algorithm = AWS_SCA_INIT; algorithm <= AWS_SCA_END; ++algorithm) {
+        char object_path_sprintf_buffer[128] = "";
+        snprintf(
+            object_path_sprintf_buffer,
+            sizeof(object_path_sprintf_buffer),
+            "/prefix/round_trip/test_default_fc_%d.txt",
```

Comment:
is this file in source control?
if it is, could we use the checksum's name instead of an int?
maybe use aws_get_http_header_name_from_algorithm() to get the string?